cardano-node/src/Cardano/Node/Configuration/POM.hs

```diff
@@ -242,6 +243,7 @@ instance FromJSON PartialNodeConfiguration where
 
       -- Node parameters, not protocol-specific
       pncSocketPath <- Last <$> v .:? "SocketPath"
+      pncDatabaseFile <- Last <$> v .:? "DatabaseFile"
```

Comment:
I'm unsure why we call this "...File" when it is a folder, perhaps we should rename it to "...Path" or "...Dir"

Reply:
"DatabasePath" it is